src/main/java/xyz/vegaone/easytracking/service/SprintService.java

```diff
@@ -0,0 +1,81 @@
+package xyz.vegaone.easytracking.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import xyz.vegaone.easytracking.domain.SprintEntity;
+import xyz.vegaone.easytracking.dto.Sprint;
+import xyz.vegaone.easytracking.mapper.SprintMapper;
+import xyz.vegaone.easytracking.mapper.UserStoryMapper;
+import xyz.vegaone.easytracking.repo.SprintRepo;
+import xyz.vegaone.easytracking.repo.UserStoryRepo;
+
+import java.util.Collections;
+import java.util.Optional;
+
+@Service
+public class SprintService {
+
+    private SprintRepo sprintRepo;
+
+    private SprintMapper sprintMapper;
+
+    private UserStoryService userStoryService;
+
+    private UserStoryRepo userStoryRepo;
+
+    private UserStoryMapper userStoryMapper;
+
+    @Autowired
+    public SprintService(SprintRepo sprintRepo, SprintMapper sprintMapper, UserStoryService userStoryService, UserStoryRepo userStoryRepo, UserStoryMapper userStoryMapper) {
+        this.sprintRepo = sprintRepo;
+        this.sprintMapper = sprintMapper;
+        this.userStoryService = userStoryService;
+        this.userStoryRepo = userStoryRepo;
+        this.userStoryMapper = userStoryMapper;
+    }
+
+
+
+    public Sprint createSprint(Sprint sprint){
+        SprintEntity sprintEntity = sprintMapper.dtoToDomain(sprint);
+        SprintEntity savedSprintEntity = sprintRepo.save(sprintEntity);
+        Sprint savedSprint = sprintMapper.domainToDto(savedSprintEntity);
+
+        Long sprintNumber = savedSprintEntity.getId() + 1;
```

Comment:
`sprintNumber` is created but never used. We'll need a way to assign the sprint number. I'll have to think about it.

Reply:
@sabaubogdan remove this line ` Long sprintNumber = savedSprintEntity.getId() + 1;`. Add a `TODO: figure out a way to generate sprint numbers` And merge this pull request